docs/faq/setup.md

```diff
@@ -381,4 +381,108 @@ F1121 15:21:15.154526 3671032 certmanager.go:94] Error: failed to get CA certifi
 
 One more important thing about `cloudcore` container mode is about how to expose cloudcore port to edge nodes. In container mode, we will also create a cloudcore service. And it's your duty to choose a LoadBalancer or adjust it to `NodePort` ServiceType, to expose `cloudcore` service to edge nodes. For more details, please reference [k8s service docs](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
 
+## Runtime related issues
```

Comment:
How about moving the `Runtime related issues` to the top? As there're many issues releted to them.

Reply:
> How about moving the `Runtime related issues` to the top? As there're many issues releted to them.

 I find the preceding content is also significant, so I placed it here. Users can use the directory on the right to quickly locate it.   But this directory can be further organized for better clarity and ease of use.

![image](https://github.com/kubeedge/website/assets/48788150/f9835b2c-d37a-4dc0-a914-e0938f137e40)


Alternatively, we can add subdirectories below the setup, resulting in the following effect. 

![image](https://github.com/kubeedge/website/assets/48788150/9e481af7-c15f-4706-89db-5cc41d0be30d)

Reply:
subdirectories looks great.